src/main/resources/user.csv

```diff
@@ -1,2 +1,2 @@
 login,password
-test,test
\ No newline at end of file
+Yevhen_Hrafov@epam.com,SUgrafin6353
```

Comment:
This file should be hidden because there is your password and login here. You must add it to .gitignore